src/scenes/movieScene/MovieScene.test.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { MovieScene, mapStateToProps, mapDispatchToProps } from './MovieScene';
+import { shallow } from 'enzyme';
+
+describe('MovieScene wrapper component', () => {
+    const movies = [
+        {
+            genres: ["Action"],
+            id: 1,
+        },
+        {
+            genres: ["Test"],
+            id: 2,
+        }
+    ];
+    const movie = movies[0];
+
+    const initialState = {
+        movies: {
+            data: movies,
+            movie
+        }
+    };
+
+    const match = {
+        params: {
+            id: 1
+        }
+    };
+
+    it('mapStateToProps should return appropriate props', () => {
+        const expectedResult = {movies, movie};
+        const receivedResult = mapStateToProps(initialState);
+        expect(receivedResult).toEqual(expectedResult);
+    });
+
+    it('mapDispatchToProps should return appropriate props', () => {
+        const dispatch = jest.fn();
+        const props = mapDispatchToProps(dispatch);
+        const id = 44;
+        props.getMovie(id);
+        expect(dispatch).toHaveBeenCalled();
+    });      
```

Comment:
I would say testing `mapStateToProps` and `mapDispatchToProps` (and exporting it for the matter) is a little bit excessive. It's debatable, not like testing `connect` itself, but pretty close. Especially if these functions are doing something very obvious.
But good you've found one of the ways to test them though.